<!-- This is an example launch file for how a ROS application could use a node -->
<launch>

	<arg name="instance_id" default="default"/>
	<param name="instance_id" value="$(arg instance_id)"/>
    <rosparam file="$(find harmoni_detcustom)/config/configuration.yaml" subst_value="True"/>
    <node pkg="harmoni_detcustom" type="detcustom_service.py" name="harmoni_detcustom_$(arg instance_id)" output="screen"/>

</launch>
